README.md

```diff
@@ -13,10 +13,10 @@ eslint sharable config for semistandard
 ## Install
 
 ```bash
-npm install eslint-plugin-promise eslint-plugin-standard eslint-plugin-react
-npm install eslint-config-standard
-npm install eslint-config-semistandard
-# note that eslint-plugin-promise, eslint-plugin-standard, & eslint-plugin-react are required peer dependencies
+npm install --save-dev eslint-plugin-promise eslint-plugin-standard eslint-plugin-node eslint-plugin-import
+npm install --save-dev eslint-config-standard
+npm install --save-dev eslint-config-semistandard
+# note that eslint-plugin-promise, eslint-plugin-standard, & eslint-plugin-node eslint-plugin-import are required peer dependencies
```

Comment:
Plus `eslint-config-standard` which gets extended by this one. Please add it to the list and adjust commas and & position inside sentence👍

Reply:
Done

Reply:
Merci 👍